client/src/components/Slack.js

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
 
-import slackLogo from '../images/Slack_Mark_Web.png'
+import slackLogo from '../assets/images/Slack_Mark_Web.png'
 
```

Comment:
Please move the tbot image into the assets folder and update the import on the no-match component, since we have a new assets folder.  Go ahead and delete the image folder once you have moved it as well.